tests/_data/models.py

```diff
@@ -1437,6 +1437,26 @@ def get_bom_for_issue540_duplicate_components() -> Bom:
     bom.register_dependency(component1, [component3])
     return bom
 
+
+def get_bom_for_issue799_organizational_entity_bom_ref() -> Bom:
+    """regression test for issue #799
+    see https://github.com/CycloneDX/cyclonedx-python-lib/issues/799
+    """
+    return _make_bom(
```

Comment:
please don't use the `$.metadata.tools.services` for this use case, as this particular structure is not available in all versions of CycloneDX, as this would render this test case almost unused.


please use `$.components[].supplier` instead.